discord/ext/modal_paginator/core.py

```diff
@@ -502,6 +513,12 @@ def from_text_inputs(
             The constructed paginator with the modals.
         """
 
+        if titles_steps < 0 or ((titles_steps > len(titles)) if not isinstance(titles, str) else True):
+            raise ValueError('titles_step must be between 0 and the length of the titles, if available')
```

Comment:
Could mention the length
```suggestion
            raise ValueError('titles_step must be between 0 and the length of the titles ({len(titles)}), if available')
```